packages/account/src/Components/currency-selector/__tests__/currency-selector.spec.tsx

```diff
@@ -241,78 +63,309 @@ describe('<CurrencySelector/>', () => {
         expect(screen.getByText(msg)).toBeInTheDocument();
         expect(screen.getByRole('button', { name: /next/i })).toBeEnabled();
     };
+    const store = mockStore({
+        client: {
+            accounts: {
```

Comment:
maybe we should add such account example to mockStore itself and reuse it in many places?

Reply:
It might look beneficial  to Account package testing. But might effect testing of other package as we do not know the particular account config that the testcase is written on